Src/LogData.c

```diff
@@ -250,32 +267,32 @@ void updateLogEntry(AllData* data)
 void logEntryOnceRoutine(AllData* data)
 {
     updateLogEntry(data);
-    writeLogEntryToEEPROM(currMemAddress);
+    writeLogEntryToEEPROM(EEPROM_START_ADDRESS + logAddressOffset);
+    logAddressOffset += LogEntrySize;
 }
 
 void logDataTask(void const* arg)
 {
     AllData* data = (AllData*) arg;
+    //TODO: make log entry here and pass it around (LogEntry log;) instead of a global var?
```

Comment:
Yup! Global variables aren't always a good idea, so something like this would be better:
```c
LogEntry log;
initializeLogEntry(&log);
```